Project-Aurora/Project-Aurora/Project-Aurora.csproj

```diff
@@ -102,6 +102,9 @@
     <Reference Include="HidLibrary, Version=3.2.46.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\hidlibrary.3.2.46.0\lib\HidLibrary.dll</HintPath>
     </Reference>
+    <Reference Include="HidSharp, Version=2.1.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Howcome we have a dependency to HidSharp now?

Reply:
That's what Vulcan.net uses to interface with the keyboard. Hidlibrary is too slow. Snakepin suggested we transition unifiedhid over to hidsharp as well

Reply:
Correct me if I'm wrong, but shouldn't the fact that it's a dependency of Vulcan.NET mean that the dependency is on Vulcan rather than Aurora?